src/app/organization/tellers/view-cashier/view-cashier.component.html

```diff
@@ -0,0 +1,67 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary" [routerLink]="['edit']">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+</div>
+
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <h2 class="mat-h2" fxFlexFill>{{ cashierData.staffName}}</h2>
+
+      <mat-divider [inset]="true"></mat-divider>
+
+      <div fxLayout="row wrap" class="content">
+
+        <div fxFlex="50%" class="header">
+          Teller
+        </div>
+
+        <div fxFlex="50%">
+          {{ cashierData.tellerName }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Description
+        </div>
+
+        <div fxFlex="50%">
```

Comment:
make it conditional if data not present

Reply:
You mean if the value is empty, the Description and its value both should be removed?